.github/workflows/ci.yml

```diff
@@ -38,6 +43,8 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
+      #- name: "Install KeywordCalls branch"
+      #  run: julia --project=. -e 'using Pkg; Pkg.add(PackageSpec(name="KeywordCalls", rev="master"))'
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
```

Comment:
I think all the changes here can be reverted?

Reply:
c7b173a